lib/ansible/modules/clustering/k8s/k8s.py

```diff
@@ -124,6 +124,15 @@
       the generated hash and append_hash=no)
     type: bool
     version_added: "2.8"
+  apply:
+    description:
+    - Whether to use 'apply' method to update resources.
+    - C(apply) remembers previous invocations of k8s apply through an annotation"
+    - C(apply) attempts to ensure a resource is exactly as specified - it's closer to a replace than a patch."
```

Comment:
I would avoid the `closer to replace` phrasing, it can do pruning but is still sending a `PATCH` (just an intelligently computed `PATCH` instead of the hail mary we're currently sending). If possible I'd try to match our phrasing to the [official docs](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#kubectl-apply) as much as possible

Reply:
Ok, I've added a new commit that updates those docs, can you see if that's better?